examples/dynamic-imports/static.config.js

```diff
@@ -36,4 +58,61 @@ export default {
       },
     ]
   },
+  renderToHtml: (renderToString, App, meta, prodStats) => {
+    const chunkNames = []
+    const appHtml = renderToString(
+      <ReportChunks report={chunkName => chunkNames.push(chunkName)}>
+        <App />
+      </ReportChunks>,
+    )
+
+    const { scripts } = flushChunks(prodStats, {
+      chunkNames,
+    })
+
+    meta.scripts = scripts.filter(script => script.split('.')[0] !== 'app')
+    return appHtml
+  },
+  Document: ({ Html, Head, Body, children, siteProps, renderMeta }) => (
+    <Html lang="en-US">
+      <Head>
+        <meta charSet="UTF-8" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0"
+        />
+      </Head>
+      <Body className="slug-home">
+        {children}
+        {renderMeta.scripts &&
+          renderMeta.scripts.map(script => <script type="text/javascript" src={`/${script}`} />)}
+      </Body>
+    </Html>
+  ),
+  webpack: (config, { stage, defaultLoaders }) => {
+    if (stage === 'node') {
+      config.externals = externals
+
+      config.plugins.push(
+        new webpack.optimize.LimitChunkCountPlugin({
```

Comment:
Do you all see an issue with always doing this?  If not, we could bake a bunch of the Webpack config into core and nobody would really notice